bin/node/runtime/Cargo.toml

```diff
@@ -70,8 +70,6 @@ pallet-indices = { version = "4.0.0-dev", default-features = false, path = "../.
 pallet-identity = { version = "4.0.0-dev", default-features = false, path = "../../../frame/identity" }
 pallet-membership = { version = "4.0.0-dev", default-features = false, path = "../../../frame/membership" }
 pallet-multisig = { version = "4.0.0-dev", default-features = false, path = "../../../frame/multisig" }
-pallet-nomination-pools = { version = "1.0.0", default-features = false, path = "../../../frame/nomination-pools"}
```

Comment:
Is it by intention?

Reply:
There is the dependency deletion, but no pallet configuration deletion in `lib.rs` by the way.

Reply:
Yes. Pallet configuration deletion has been executed in the previous version.